google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/AddGoogleMyBusinessLocationExtensions.java

```diff
@@ -142,6 +147,10 @@ private void runExample(
                 PlacesLocationFeedData.newBuilder()
                     .setEmailAddress(StringValue.of(gmbEmailAddress))
                     .setBusinessAccountId(StringValue.of(businessAccountIdentifier))
+                    // Used to filter Google My Business listings by labels. If entries exist in
+                    // label_filters, only listings that has any of the labels set are candidates
```

Comment:
nit: "only listings that has any of the labels" -> "only listings that **have** at least one of the labels"